README.md

```diff
@@ -2,13 +2,13 @@
 
 # Azure Management Libraries for Java
 
-This README is based on the released stable version (1.1.0). If you are looking for other releases, see [More Information](#more-information).
+This README is based on the released stable version (1.2.0). If you are looking for other releases, see [More Information](#more-information).
 
 The Azure Management Libraries for Java is a higher-level, object-oriented API for managing Azure resources. If you are looking for Java client libraries for Azure services, please see [Azure libraries for Java](https://docs.microsoft.com/en-us/java/azure/java-sdk-azure-install) - this page will be maintained regularly to ensure that it is up to date.
 
 Note: This page will be maintained regularly to ensure that it is up to date.
 
-## Feature Availability and Road Map as of Version 1.1.0 ##
+## Feature Availability and Road Map as of Version 1.2.0 ##
 
 <table>
```

Comment:
@selvasingh Why do we do a HTML table here instead of a markdown table? Just curious.